GoogleCloudPlatform/microservices-demo
# Update purpose.md to relate to the retail industry

### Background 
* Please see [private Google Doc here](https://docs.google.com/document/d/1KU66QLgpIb2IpSyHUW7fHDrceGH5Y0SEAOexiY1Si4c/edit?usp=sharing&resourcekey=0-cvTBMv8pVLc0DGOcnHy50w).

### Change Summary
* Update the purpose of Online Boutique to include relevance to the retail industry.

### Testing Procedure
* As a reviewer, please:
   * check grammar/word choice/spelling
   * readability of `purpose.md`
   * see if changing the purpose is a good idea

### Related PRs or Issues 
* https://github.com/GoogleCloudPlatform/microservices-demo/issues/1590
